devauntea's Poem

```diff
@@ -0,0 +1,14 @@
+copy rat
+They say copying is the highest form of flattery
+but i think its because you have no originality
```

Comment:
Question: Did you mean for the "i" to be lowercase?

Reply:
no